### DISCUSSION TITLE
How to train a custom NER and preserve the standard entity classification also

### DISCUSSION THREAD

**Discussion by DarkSoliditi** [explosion/spaCy#9344] [Help: Coding & Implementations] [answered]
I was training a custom ner model and it's working great but I was wondering if there was a way to preserve spacy's out of the box entity classifications. I tried to train it with en_core_web_trf loaded instead of spacy.blank('en') and thought it would retain the original classification.

Any help is greatly appreciated

**Comment by polm** [comment, marked as answer]
You can load multiple NER component in the same pipeline, see the [double NER tutorial](https://github.com/explosion/projects/tree/v3/tutorials/ner_double).

**Reply by DarkSoliditi** [reply]
That works perfectly! Thanks so much!!
